packages/web/components/Dashboard/Calendar.tsx

```diff
@@ -247,100 +240,103 @@ export const Calendar: React.FC = () => {
-                                          event.end.dateTime,
-                                        ).toLocaleString(
-                                          DateTime.TIME_24_SIMPLE,
-                                        )
-                                      : DateTime.fromISO(
-                                          event.end.date,
-                                        ).toLocaleString(
-                                          DateTime.TIME_24_SIMPLE,
-                                        )
-                                  }
-                                  description={cleanDescription(
-                                    event.description,
-                                  )}
-                                  htmlLink={event.htmlLink}
-                                  location={event.location}
-                                  addToCalUrl={buildAddToCalendarLink(event)}
-                                />
-                              </Popover>
+                            return (
                               <Box
-                                position="absolute"
-                                inset={0}
+                                as="li"
+                                position="relative"
+                                className="calendar__day--event"
+                                width="100%"
+                                px={5}
+                                py={2}
+                                backgroundColor="blackAlpha.500"
+                                borderRadius="md"
+                                overflow="hidden"
+                                role="group"
                                 _groupHover={{
                                   backgroundColor: 'blackAlpha.600',
                                 }}
-                                transition="all 0.1s ease-in-out"
-                                zIndex={-1}
-                              />
-                            </Box>
-                          ))}
+                              >
+                                <Popover colorScheme="purple">
+                                  <Event
+                                    title={event.summary}
+                                    start={
+                                      'dateTime' in event.start
+                                        ? DateTime.fromISO(
+                                            event.start.dateTime,
+                                          ).toLocaleString(
+                                            DateTime.TIME_24_SIMPLE,
+                                          )
+                                        : DateTime.fromISO(
+                                            event.start.date,
+                                          ).toLocaleString(
+                                            DateTime.TIME_24_SIMPLE,
+                                          )
+                                    }
```

Comment:
Will this work?

```suggestion
                                    start={
                                      DateTime.fromISO(
                                        event.start.dateTime ?? event.start.date,
                                      ).toLocaleString(
                                        DateTime.TIME_24_SIMPLE,
                                      )
                                    }
```

Reply:
I get `Property 'dateTime' does not exist on type 'GoogleCalEventDateTimeType'.
  Property 'dateTime' does not exist on type '{ date: string; }'`

And the type defined in `useCalendar`` is: 
```
export type GoogleCalEventDateTimeType =
  | {
      date: string;
    }
  | {
      dateTime: string;
      timeZone: string;
    };
    ``````

Reply:
I didn't think about TypeScript. You could still do:

```js
start={
  DateTime.fromISO(
    'dateTime' in event.start ? event.start.dateTime : event.start.date,
  ).toLocaleString(
    DateTime.TIME_24_SIMPLE,
  )
}
```